### PR TITLE
Docs: adding CKBOx editing to demos

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: adding CKBOx editing to demos. Closes https://github.com/ckeditor/ckeditor5/issues/15225

---

### Additional information

_Merge together with https://github.com/cksource/ckeditor5-commercial/pull/5664_

Image Editing was added to existing Image Contextual Toolbars only. Make a follow-up for adding missing Image Contextual Toolbars in the docs.

### CODE REVIEW

**Review by mmotyczynska — CHANGES_REQUESTED** [review]
There are some questions and suggestions in the comments.

Besides that, I did not find the `Edit Image` button in the following features (I did not check all of them, but it turns out it is because of the missing image toolbar config or used Easy Image). I believe this will be covered in a follow-up ticket. But I am pasting the list here anyway for future reference:

- Autoformatting
- Automatic text transformation
- Autosave
- Basic text styles
- Block indentation
- Code blocks
- Content minimap
- Editor placeholder
- Find and replace
- Font family, size, and color
- Highlight
- General HTML Support
- HTML comment element
- Full page HTML
- Link
- To-do lists
- Media embed
- Mentions
- Paste plain text
- Read-only support
- Remove formatting
- Restricted editing
- Select all
- Show blocks
- Spelling, grammar, and punctuation checking
- Styles
- Text alignment
- Text part language
- Setting the UI language

And for the Images (I think some of them can be done in this ticket):

- Image text alternative
- Image captions
- Resizing images
- Responsive images
- Image styles
- Linking images
- Inserting images via URL (missing image toolbar)

File: docs/_snippets/features/toolbar-nested-label.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ ClassicEditor
 				items: [ 'bold', 'italic', 'strikethrough', 'superscript', 'subscript' ]
 			}
 		],
+		image: {
+			toolbar: [ 'imageStyle:inline', 'imageStyle:block', 'imageStyle:side',
+				'|', 'toggleImageCaption', 'imageTextAlternative', '|', 'ckboxImageEdit' ]
+		},
```
[the remark this answers is not in the document]
Comment (gorzelinski) [reply docs/_snippets/features/toolbar-nested-label.js:30]: I tested it. If we want to keep the functionality of pasting images, it makes sense to leave the config.

File: packages/ckeditor5-table/docs/_snippets/features/table-basic.js [hunk 2]
```diff
@@ -17,7 +17,9 @@ ClassicEditor
 				'imageStyle:side',
 				'|',
 				'toggleImageCaption',
-				'imageTextAlternative'
+				'imageTextAlternative',
+				'|',
+				'ckboxImageEdit'
```
[the remark this answers is not in the document]
Comment (gorzelinski) [reply packages/ckeditor5-table/docs/_snippets/features/table-basic.js:22]: It seems that no page is using this snippet. We probably don't need it.

File: packages/ckeditor5-word-count/docs/_snippets/features/build-word-count-source.js [hunk 3]
```diff
@@ -92,7 +92,9 @@ BalloonEditor.defaultConfig = {
 			'imageStyle:side',
 			'|',
 			'toggleImageCaption',
-			'imageTextAlternative'
+			'imageTextAlternative',
+			'|',
+			'ckboxImageEdit'
```
[the remark this answers is not in the document]
Comment (gorzelinski) [reply packages/ckeditor5-word-count/docs/_snippets/features/build-word-count-source.js:97]: Yeah, these snippets have their configs. The config in the base snippet is probably redundant.

**Review by mmotyczynska — APPROVED** [review]
LGTM 👍

**Comment by godai78** [comment]
Actually I left out the image demos on purpose. They each highlight a single toolbar item, so throwing edit into each one seemed like a bad idea.

**Pull request merged**